### ISSUE TITLE
Breakfast Club #5: You only have breakfast twice #iexpectyoutocaffeinate

### ISSUE THREAD

**Issue by javorszky** [breakfast-club-oxford/event-organisation#5]
Venue: [Ye Olde Firestation Cafe](https://oldfirestation.org.uk/shop-cafe/cafe/)
When: June 2nd, 8:30am - 10:00am. 
---

This one opens later than usual, which would allow some of the later rising folk to join us as well. The venue is quite chilled, and all proceeds go to helping the local community, which is a double win!

As usual, we'll lurk in the #breakfastclub channel over yonder slack.

**Comment by javorszky** [comment]
We've had a nice turnout, Gordon, Beverly, Pedro, Iulian, and myself made it. There was a grand total of 1 tie present (worn by Iulian).

We've talked about how book publishers are super bad at using git, and [gitbook](gitbook.com) later. Beverly had the experience of what happens when testing is done on production.
![8054 image_thumb_35c6e986](https://cloud.githubusercontent.com/assets/1196299/26720170/199f7298-477f-11e7-8bb6-4dac33ea7c3b.png)

Later Iulian arrived and brought the scalp of Otto:

![2017-06-02 09 47 39](https://cloud.githubusercontent.com/assets/1196299/26721178/dbfa5f76-4782-11e7-8231-965d81ff291f.jpg)

We then talked about ultrasound sensors, and [laser distance modules](www.espruino.com/VL53L0X)

![2017-06-02 09 50 09](https://cloud.githubusercontent.com/assets/1196299/26721277/39693862-4783-11e7-8a18-908dfcb9ec8d.jpg)

![2017-06-02 09 26 18](https://cloud.githubusercontent.com/assets/1196299/26721372/8060c0b4-4783-11e7-8b28-3b0e87baedb3.jpg)

All in all, good effort from everyone! :)
